packages/ui/useScroll/useScroll.js

```diff
@@ -1,3 +1,16 @@
+/**
+ * Unstyled scrollable hook and behaviors to be customized by a theme or application.
+ *
+ * @module ui/useScroll
+ * @exports constants
+ * @exports Scrollable
```

Comment:
If we don't need, we could not export the `Scrollable`.

Reply:
Let's remove the ui/useScroll.Scrollable.js.

Reply:
I'll do after removing the references from other theme libs.